packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -258,35 +258,20 @@ export class Meter implements api.Meter {
 
   /**
    * Creates a new batch observer metric.
-   * @param name the name of the metric.
    * @param callback the batch observer callback
    * @param [options] the metric batch options.
    */
   createBatchObserver(
-    name: string,
     callback: (observerResult: api.BatchObserverResult) => void,
     options: api.BatchMetricOptions = {}
```

Comment:
should the `api.BatchMetricOptions` not inherit from `MetricOptions` now ?
Perhaps the `api.BatchMetricOptions` should be renamed to `api.BatchObserverOptions` ?